zilliqa/src/p2p_node.rs

```diff
@@ -28,13 +29,15 @@ use crate::{
     cfg::{Config, ConsensusConfig, NodeConfig},
     crypto::SecretKey,
     message::{ExternalMessage, InternalMessage},
-    networking::{request_response, MessageCodec, MessageProtocol, ProtocolSupport},
     node_launcher::NodeLauncher,
 };
 
+/// Messages are a tuple of the destination shard ID and the actual message.
+type P2pMessage = (u64, ExternalMessage);
```

Comment:
These are _only_ used for the RequestResponse behaviour, right? I think it'd be better to name it something more descriptive like `P2pDirectMessage` for example, because `P2pMessage` sounds like it's the general form of p2p communication including gossipsub broadcasts, but that's not true if it's only used for direct request/response and not gossipsub.

Reply:
Thank you! Yes, "Direct" is much clearer for what I was trying to say.